### DISCUSSION TITLE
The Hall of Firsts

### DISCUSSION THREAD

**Discussion by Psypher9** [rezalas/riftshadow#194] [Ideas]
It could be cool to have an area that is like a Museum of the "Firsts" to do a thing.

For example:

- The first character to be a Paladin
- The first to find a secret area
- The first to defeat a certain mob

walking around the area would have plaques to read about who the character was and what they did, as well as when it happened.

**Comment by rezalas** [comment]
I really like this! We could track the firsts in SQL as well and cache it on startup maybe?

**Comment by Psypher9** [comment]
Thinking about this again, I'm not as familiar with how something gets made in-world.

Is the process to do that known?

**Reply by rezalas** [reply]
Yes, the easiest way is to use the in-game system for it actually. Once an admin promotes you past hero to Avatar or higher I believe you can use the `dig` command which lets you create new rooms and develop new areas of the game.
